CONTRIBUTING.md

```diff
@@ -0,0 +1,370 @@
+    documentation](http://docs.scipy.org/doc/numpy/dev/index.html).
+-   Matthew Brett's
+    [Pydagogue](http://matthew-brett.github.com/pydagogue/).
+
+### Getting Started with Git
+
+[GitHub has instructions](http://help.github.com/set-up-git-redirect)
+for installing git, setting up your SSH key, and configuring git. All
+these steps need to be completed before working seamlessly with your
+local repository and GitHub.
+
+### Forking
+
+You may want to fork UV-CDAT to work on the code if you have access to the repository,
+then just create a branch there instead. If you don't then follow these guidelines
+for forking UV-CDAT. Go to the [UV-CDAT project page](https://github.com/UV-CDAT/uvcdat)
+and hit the *fork* button. You will want to clone your fork to your machine: (HTTPS
+or SSH is preferred to git:// for security reasons).
+
+    git clone git://github.com/UV-CDAT/uvcdat.git UV-CDAT-yourname
+    cd UV-CDAT-yourname
+    git remote add myuvcdat git@github.com:your-user-name/uvcdat.git
+
+This creates the directory UV-CDAT-yourname and connects your repository
+to the upstream (main project) *UV-CDAT* repository.
+
+You will also need to hook up Travis-CI to your GitHub (if you have forked)
+repository so the suite is automatically run when a Pull Request is submitted.
+Instructions are [here](http://about.travis-ci.org/docs/user/getting-started/).
+
+### Creating a Branch
+
+You want your master branch to reflect only production-ready code, so
+create a feature branch for making your changes. For example:
+
+    git branch shiny-new-feature
+    git checkout shiny-new-feature
+
+The above can be simplified to:
+
+    git checkout -b shiny-new-feature
+
+This changes your working directory to the shiny-new-feature branch.
+Keep any changes in this branch specific to one bug or feature so it is
+clear what the branch brings to *UV-CDAT*. You can have many
+shiny-new-features and switch in between them using the git checkout
+command.
+
+### Making changes
+Before making your code changes, it is often necessary to build the code
+that was just checked out. The best way to develop *UV-CDAT* is to build
+using default settings:
+
+        mkdir uvcdat-build
+        cmake uvcdat-path-to-source
+        make -jN
+
+Contributing to the documentation
+---------------------------------
+
+If you're not the developer type, contributing to the documentation is
+still of huge value. You don't even have to be an expert on *UV-CDAT* to
+do so! Something as simple as pointing missing information or broken links
+will be of great value.
+
+Contributing to the code base
+-----------------------------
+### Code Standards
+
+*UV-CDAT* uses the [PEP8](http://www.python.org/dev/peps/pep-0008/)
+standard. There are several tools to ensure you abide by this standard.
+
+Alternatively, use [flake8](http://pypi.python.org/pypi/flake8) tool for
+checking the style of your code. Additional standards are outlined on
+the [code style wiki
+page](LINK HERE).
+
```

Comment:
I think we should. I just left it like that for the future but I can take it out for now.